src/arrayMethodSort.js

```diff
@@ -3,9 +3,38 @@
 /**
  * Implement method Sort
  */
+const defaultCompareFunction = (a, b) => {
+  if (a.toString() < b.toString()) {
+    return -1;
+  }
+
+  if (a.toString() > b.toString()) {
+    return 1;
```

Comment:
will be better to convert values to string before all checks one time for not repeating your code